sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -65,6 +65,7 @@ case object StoreTaskCompletionListener extends RocksDBOpType("store_task_comple
  * @param localRootDir Root directory in local disk that is used to working and checkpointing dirs
  * @param hadoopConf   Hadoop configuration for talking to the remote file system
  * @param loggingId    Id that will be prepended in logs for isolating concurrent RocksDBs
+ * @param providerListener A reference to the state store provider for event callback reporting
```

Comment:
nit: confusing comment? I mean the "A reference to the state store provider" part

Reply:
Hmm okay, will change that part to "The parent RocksDBStateStoreProvider object" to help make it more clear :+1: